src/main/resources/optifabric.optifine.mixins.json

```diff
@@ -5,9 +5,8 @@
   "mixins": [
     "MixinReflectorClass",
     "MixinScreen",
-    "MixinClientBuiltinResourcePackProvider",
     "MixinOptifineConfig",
-    "MixinGameOptions"
+    "MixinDefaultResourcePack"
```

Comment:
This is duplicated in `optifabric.mixins.json`; I think Mixin will just ignore it, but probably best that it's only in the one